code/modules/surgery/generic.dm

```diff
@@ -3,243 +3,283 @@
+/datum/surgery_step/generic/cauterize/begin_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/obj/item/organ/external/affected = target.get_organ(target_zone)
-	user.visible_message("[user] is beginning to cauterize the incision on [target]'s [affected.name] with \the [tool]." , \
-	"You are beginning to cauterize the incision on [target]'s [affected.name] with \the [tool].")
+	user.visible_message(
+		"[user] is beginning to cauterize the incision on [target]'s [affected.name] with \the [tool].",
+		"You are beginning to cauterize the incision on [target]'s [affected.name] with \the [tool]."
+	)
 	target.custom_pain("Your [affected.name] is being burned!")
-	..()
+	return ..()
 
-/datum/surgery_step/generic/cauterize/end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool,datum/surgery/surgery)
+/datum/surgery_step/generic/cauterize/end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/obj/item/organ/external/affected = target.get_organ(target_zone)
-	user.visible_message("<span class='notice'> [user] cauterizes the incision on [target]'s [affected.name] with \the [tool].</span>", \
-	"<span class='notice'> You cauterize the incision on [target]'s [affected.name] with \the [tool].</span>")
-	affected.open = 0
+	user.visible_message(
+		"<span class='notice'>[user] cauterizes the incision on [target]'s [affected.name] with \the [tool].</span>",
+		"<span class='notice'>You cauterize the incision on [target]'s [affected.name] with \the [tool].</span>"
+	)
+	affected.open = ORGAN_CLOSED
 	affected.germ_level = 0
-	return TRUE
+	return SURGERY_STEP_CONTINUE
 
-/datum/surgery_step/generic/cauterize/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool,datum/surgery/surgery)
+/datum/surgery_step/generic/cauterize/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/obj/item/organ/external/affected = target.get_organ(target_zone)
-	user.visible_message("<span class='warning'> [user]'s hand slips, leaving a small burn on [target]'s [affected.name] with \the [tool]!</span>", \
-	"<span class='warning'> Your hand slips, leaving a small burn on [target]'s [affected.name] with \the [tool]!</span>")
+	user.visible_message(
+		"<span class='warning'>[user]'s hand slips, leaving a small burn on [target]'s [affected.name] with \the [tool]!</span>",
+		"<span class='warning'>Your hand slips, leaving a small burn on [target]'s [affected.name] with \the [tool]!</span>"
+	)
 	target.apply_damage(3, BURN, affected)
-	return FALSE
+	return SURGERY_STEP_RETRY
+
+/datum/surgery_step/generic/cauterize/premature
+	name = "cauterize incision (premature)"
+
+/datum/surgery_step/generic/cauterize/premature/begin_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	. = ..()
```

Comment:
Is this `. = ..()` meant to be here, considering there is a `return ..()` at the end of the proc?

Reply:
Might have just missed that when I was making sure they all returned ..()